mentorship ios/Service/Networking/SignUpAPI.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  SignUpAPI.swift
+//  Created on 23/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Foundation
+import Combine
+
+class SignUpAPI: SignUpService {
+    private var cancellable: AnyCancellable?
+    
+    func signUp(
+        availabilityPickerSelection: Int,
+        signUpData: SignUpModel.SignUpUploadData,
+        confirmPassword: String,
+        completion: @escaping (SignUpModel.SignUpResponseData) -> Void
+    ) {
+        // make variable for sign up data
+        var signUpData = signUpData
+        
+        //assign availability values as per picker selection
+        if availabilityPickerSelection > 1 {
+            signUpData.needMentoring = true
+        }
+        if availabilityPickerSelection != 2 {
```

Comment:
can we do this:
```
signUpData.availableToMentor = availabilityPickerSelection != 2

Reply:
This is incredibly neat! Thanks! Will save an if condition too.